src/api_manager/service_control/aggregated.h

```diff
@@ -183,6 +183,9 @@ class Aggregated : public Interface {
   int check_retries_;
   int report_retries_;
   int quota_retries_;
+
+  // network fail policy, default to false
+  bool network_fail_open_{};
```

Comment:
delete {}?

Reply:
I want to initialize the value as false, even it is done so in Init() call. Even Init() is always called, but by putting {} here is to make sure it is initialized as false without adding lines to two constructors.